packages/blockchain-wallet-v4-frontend/src/data/components/manageAddresses/actions.js

```diff
@@ -1,92 +1,122 @@
 import * as AT from './actionTypes'
 
 // UNUSED ADDRESSES
-export const deleteAddressLabel = (accountIdx, walletIdx, addressIdx) => ({
+export const deleteAddressLabel = (
```

Comment:
Just wondering if we need to pass derivation to all these functions @schnogz or can we figure that out deeper down in the core?

Reply:
Nevermind, we need to be explicit here, you're right